tools/runners/staking_runner.py

```diff
@@ -124,14 +153,45 @@ def resume_staking_contracts():
         count += 1
 
 
+def resume_staking_contract(staking_address: str):
+    """Resume staking contract"""
+
+    print("Resume staking contract")
+
+    network_providers = NetworkProviders(API, PROXY)
+    dex_owner = get_owner(network_providers.proxy)
+
+    if not os.path.exists(OUTPUT_PAUSE_STATES):
```

Comment:
Just as a note, I'm thinking that for such pause and resume operations on specific given addresses we may not need to have the previous state, because we may use this specific case only intentionally.
For bulk pause&resumes it makes sense to save the previous versions before upgrades to know what to resume afterwards, but for specific given cases we may just want to do it irrespective of its previous state.

Reply:
will remove it in a next PR